mongodbatlas/resource_mongodbatlas_global_cluster_config.go

```diff
@@ -106,6 +116,15 @@ func resourceMongoDBAtlasGlobalClusterCreate(ctx context.Context, d *schema.Reso
 				Db:             mn["db"].(string),
 				CustomShardKey: mn["custom_shard_key"].(string),
 			}
+
+			if isCustomShardKeyHashed, okCustomShard := mn["is_custom_shard_key_hashed"]; okCustomShard {
+				addManagedNamespace.IsCustomShardKeyHashed = pointy.Bool(isCustomShardKeyHashed.(bool))
+			}
+
+			if isShardKeyUnique, okShard := mn["is_shard_"]; okShard {
+				addManagedNamespace.IsShardKeyUnique = pointy.Bool(isShardKeyUnique.(bool))
```

Comment:
I want to assume this is always set to false by the API if not set here?

Reply:
this struct property is a boolean pointer